File: ClientApp/src/app/pickupPoints/classes/pickupPoint.ts
import { KeyValuePair } from 'src/app/shared/classes/keyValuePair'

export class PickupPoint extends KeyValuePair {
    description: string
    route: {
        id: number
        description: string
    }
    exactPoint: string
    time: string
    coordinates: string
    isActive: boolean
}

File: Features/PickupPoints/Models/PickupPoint.cs
namespace Transfers {

    public class PickupPoint {

        public int Id { get; set; }
        public string Description { get; set; }
        public string ExactPoint { get; set; }
        public string Time { get; set; }
        public string Coordinates { get; set; }
        public bool IsActive { get; set; }
        public string UserId { get; set; }

        public int RouteId { get; set; }

        public Route Route { get; set; }
    }

}